docs/cli.md

```diff
@@ -255,6 +255,15 @@ On WSL (Windows), `wslview` opens a file or directory with the default applicati
 ```console
 $ sudo apt install wslu
 ```
+If this does not work, try running:
+```console
+$ sudo apt update
+$ sudo apt install ubuntu-wsl
```

Comment:
I'm a little confused by the multiple install commands could you help me understand?

Reply:
To be completely honest, I do not exactly know what is happening here; the commands are taken straight from the [wslu wiki](https://wslutiliti.es/wslu/install.html#ubuntu). It seemed to work for the student on [Piazza](https://piazza.com/class/lbnv6k5pd0y6bz/post/432).

Reply:
Sounds good. We could also put this on hold to see if a decent number of students are facing the install issue. So far it does just seem like the one student on Piazza.

Reply:
I think the best next step would be to try this on Ubuntu-20.04 and Ubuntu-22.04.  I suspect that is the difference.  You could do it with virtualbox / vagrant.

Reply:
Looks like `wslu` might be installed by default on Ubuntu 20.04 WSL and not on Ubuntu 22.04 WSL. https://askubuntu.com/questions/1409481/why-does-xdg-open-use-wslview#comment2448974_1409481

Reply:
I took a shot at it.  Just got somebody to check the output of `lsb_release`, which I guessed.